### ISSUE TITLE
Try dropping `any`

### ISSUE THREAD

**Issue by typekev** [JustinPiquard/learning-app#2]
Great work Justin!

I'm interested in seeing how you apply your types. In practice, we don't use the `any` type, as I'm sure you already know. Let's see how you do with your types here 😉

**Comment by JustinPiquard** [comment]
Yes I know, but I have an issue to pass function in some types or interfaces it's why I put Any.
I don't know if you have some example for me to figure it out :)

**Comment by typekev** [comment]
> Yes I know, but I have an issue to pass function in some types or interfaces it's why I put Any.
> I don't know if you have some example for me to figure it out :)

This should help :+1:: https://www.typescriptlang.org/docs/handbook/functions.html
